agreement/voteTracker.go

```diff
@@ -26,9 +26,16 @@ import (
 	"github.com/algorand/go-algorand/logging/telemetryspec"
 )
 
+// SortAddress re-export this sort, which is implemented in basics, and being used by the msgp when
```

Comment:
mis-placed comment?

Reply:
Indeed I originally had it there since that's where it was necessary but moved it to `agreement/sort.go` when I realized I needed to implement some sorts from scratch. Thanks for catching it